tests/Unit/Visualization/Python/Render3D/Test_RenderBbh.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from click.testing import CliRunner
+
+from spectre.Informer import unit_test_build_path, unit_test_src_path
+from spectre.Visualization.RenderBBH import render_bbh_command
+
+
+class TestRenderBBH(unittest.TestCase):
+    def setUp(self):
+        self.vol_test_data = os.path.join(
+            unit_test_src_path(), "Visualization/Python/VolTestData.xmf"
+        )
+        self.test_dir = os.path.join(
+            unit_test_build_path(), "Visualization/Render3D/Bbh"
+        )
+        self.output_file = os.path.join(self.test_dir, "test_render_bbh.png")
+        shutil.rmtree(self.test_dir, ignore_errors=True)
+        os.makedirs(self.test_dir, exist_ok=True)
+
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
+
+    def test_render_bbh(self):
+        runner = CliRunner()
+        result = runner.invoke(
+            render_bbh_command,
+            [self.vol_test_data, "-o", self.test_dir, "-c", "Top"],
+            catch_exceptions=False,
+        )
+        self.assertEqual(result.exit_code, 0, msg=result.output)
+        self.assertTrue(os.path.exists(self.output_file), msg=result.output)
```

Comment:
Does this test work? There's no BBH data in the repo

Reply:
No, it doesn't work, it's sort of just a skeleton for if we did have bbh data in the rep with spatial ricci and lapse output

Reply:
Ok just delete it then. We should add a test that generates fake BBH data and then runs the rendering script, but we can do that after SXScon (open an issue please).